src/js/jsx/search/SearchBar.jsx

```diff
@@ -216,7 +215,8 @@ define(function (require, exports, module) {
          */
         _updateDatalistInput: function (id) {
```

Comment:
It's super, super confusing that this variable is called `id` when it is actually an array of `id`s.